modules/ROOT/partials/configuration/tinycomments_can_delete.adoc

```diff
@@ -21,7 +21,8 @@ The following example extends the default behavior to allow the author `<Admin u
 
```

Comment:
This option needs an id added for the heading at the top of the page. e.g.
```
[[tinycomments_can_delete]]
=== `tinycomments_can_delete`
```

Reply:
Done